.github/workflows/bazel.yaml

```
@@ -16,7 +16,11 @@ jobs:
         uses: actions/cache@v3
         with:
           path: "~/.cache/bazel"
-          key: bazel
+          # Update cache on every run
+          # https://github.com/actions/cache/blob/main/tips-and-workarounds.md#update-a-cache
+          key: bazel-${{ runner.os }}-${{ github.run_id }}
+          restore-keys: |
+            bazel-${{ runner.os }}
```

```suggestion
          key: ${{ runner.os }}-bazel-${{ hashFiles('.bazelversion', '.bazelrc', 'WORKSPACE', 'WORKSPACE.bazel', 'MODULE.bazel', 'Cargo.Bazel.lock') }}
          restore-keys: |
            ${{ runner.os }}-bazel-
```
based on https://github.com/actions/cache/blob/main/examples.md#---bazel ?